### DISCUSSION TITLE
Fatal error: Allowed memory size of 134217728 bytes exhausted

### DISCUSSION THREAD

**Discussion by cmukanisa** [symfony/symfony#47316] [Q&A]
### Symfony version(s) affected

5.4.11

### Description

Hi, 
I just came across this error that I don't know of origin
PLEASE HELP
MY WEB SITE IS DOWN

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in /var/www/html/project/var/cache/dev/url_generating_routes.php on line 471

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 40960 bytes) in /var/www/html/project/vendor/symfony/error-handler/DebugClassLoader.php on line 278

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 16748544 bytes) in Unknown on line 0

### How to reproduce

Hi, 
I just came across this error that I don't know of origin
PLEASE HELP
MY WEB SITE IS DOWN

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in /var/www/html/project/var/cache/dev/url_generating_routes.php on line 471

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 40960 bytes) in /var/www/html/project/vendor/symfony/error-handler/DebugClassLoader.php on line 278

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 16748544 bytes) in Unknown on line 0

**Comment by MatTheCat** [comment]
Please don’t copy/paste the same block twice in your message, and avoid screaming in upper cases!

The exception message shows your website is running with the `dev` environment, which is optimized for local development. To run your application in production you’d want to set `APP_ENV=prod` and `APP_DEBUG=0`.

See https://symfony.com/doc/5.4/deployment.html
